optimade/models/references.py

```diff
@@ -111,41 +111,35 @@ class ReferenceResourceAttributes(EntryResourceAttributes):
 
 
 class ReferenceResource(EntryResource):
-    """ The :entry:`references` entries describe bibliographic references.
-The following properties are used to provide the bibliographic details:
+    """ The `references` entries describe bibliographic references.
+    The following properties are used to provide the bibliographic details:
+
+    - **address**, **annote**, **booktitle**, **chapter**, **crossref**, **edition**, **howpublished**, **institution**, **journal**, **key**, **month**, **note**, **number**, **organization**, **pages**, **publisher**, **school**, **series**, **title**, **volume**, **year**: meanings of these properties match the [BibTeX specification](http://bibtexml.sourceforge.net/btxdoc.pdf), values are strings;
+    - **bib_type**: type of the reference, corresponding to **type** property in the BibTeX specification, value is string;
+    - **authors** and **editors**: lists of *person objects* which are dictionaries with the following keys:
+        - **name**: Full name of the person, REQUIRED.
+        - **firstname**, **lastname**: Parts of the person's name, OPTIONAL.
+    - **doi** and **url**: values are strings.
+    - **Requirements/Conventions**:
+        - **Support**: OPTIONAL support in implementations, i.e., any of the properties MAY be `null`.
+        - **Query**: Support for queries on any of these properties is OPTIONAL.
+            If supported, filters MAY support only a subset of comparison operators.
```

Comment:
```suggestion
          If supported, filters MAY support only a subset of comparison operators.
```
Again, four spaces here is too much, right? Should only be two, right?

Reply:
Yeah I think this comes from a problem with the pandoc conversion, which over-indented multi-sentence list items. Thought I'd caught them all.